### ISSUE TITLE
Host server files from folder other than "dist"

### ISSUE THREAD

**Issue by naomiyaki** [parcel-bundler/parcel#4725]
(help-wanted)(parcel 2) I'm setting up an Electron application using Parcel 2 for bundling the renderer 🚀✨

Since [electron-builder](https://github.com/electron-userland/electron-builder) uses both dist and build folders, I set up the target to use a "bundle" folder instead. It's working just fine with `parcel-build`!

However, when I use the regular `parcel` command (or `parcel serve`) to serve the files for development, they end up in a dist folder in the root directory, even though the target entry-point is in "bundle." Am I missing where this is supposed to be configured? Or does serve just work this way?

## 🎛 Configuration
**package.json**
```
  "main": "./src/main.js",
  "renderer": "./bundle/index.html",
  "targets": {
    "renderer": {
      "publicUrl": "./",
      "context": "electron-renderer"
    }
  },
  "scripts": {
    "dev": "parcel ./src/index.html --target renderer",
    "bundle": "parcel build ./src/index.html --target renderer",
  },
```

## 🤔 Expected Behavior
My understanding is that `parcel` and `parcel serve` would host files in the same directory as the target endpoint (and `parcel build`), or that there is a way to configure this.

## 😯 Current Behavior
`parcel` and `parcel serve` always host files inside a top-level dist folder.

## 🔦 Context
I'm setting up an Electron app with parcel bundling on the renderer only.

## 💻 Code Sample
```
/root
 |-- /src
 |-- /build <-- used by electron-builder
 |    |-- main.js
 |    |-- renderer.js
 |    |-- index.js
 |-- package.json
```

## 🌍 Your Environment
Software | Version(s)
-- | --
Parcel | 2.0.0-alpha.3.2
Node | v12.18.0
npm/Yarn | npm 6.14.4
Operating System | Mac OS X Mojave 10.14.6

**Comment by naomiyaki** [comment]
I've looked around in the source and noticed some fairly recent commits that add options like `distDir`. I'm guessing this is what I'm looking for - and I've answered my own question that this will be included in the next release, but please confirm if I'm missing something.

**Comment by DeMoorJasper** [comment]
@naomiyaki yes I think you can use the `--dist-dir <dir>` flag to solve this

**Comment by naomiyaki** [comment]
Thanks for your reply @DeMoorJasper ! I upgraded my app with the 2.0 beta and the `--dist-dir` flag works perfectly. 
Can't remember where I saw this but I was under the impression that a `distDir` target option was also in a recent commit, and that didn't seem to work in my tests. That said, this is a fix for the issue, so I'm closing it with this comment. Thanks again for your help and all your great work!

By the way, I haven't really posted this anywhere yet but my [Electron + Parcel 2 starter-kit is up]((https://github.com/naomiyaki/kamajii)!) and now working with the beta 📦
